src/Symfony/Bundle/FrameworkBundle/Resources/config/fragment_renderer.xml

```diff
@@ -11,7 +11,7 @@
 
     <services>
         <service id="fragment.handler" class="Symfony\Component\HttpKernel\DependencyInjection\LazyLoadingFragmentHandler">
-            <argument type="service" id="service_container" />
+            <argument /> <!-- fragment renderer locator -->
```

Comment:
@nicolas-grekas in the compiler pass, when registering the subscriber in a dummy event dispatcher to collect subscribed events and register it lazily.

however, I don't event understand why we do this: In Symfony stable releases, the ContainerAwareEventDispatcher calls the static ``getSubscribedEvents`` statically. We don't need an instance for that.